src/game/gameOverModal.scss

```diff
@@ -0,0 +1,58 @@
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: white;
+  &::before {
+    position: absolute;
+    content: "";
+    top: 0;
+    left: 15px;
+    right: 15px;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.8);
+    @media screen and (min-width: $medium-breakpoint) {
+      left: 45px;
+      right: 45px;
+    }
+  }
+  &__content {
+    max-width: 80vw;
+    max-height: 80vh;
+    z-index: 999;
+    h2 {
+      border-bottom: 5px solid white;
```

Comment:
Use color here as a variable.

Reply:
Same as up. Use hex or should i add variable?

Reply:
1) Use HEX.
2) Move it to a variable.
3) Use a variable here 😄